.github/workflows/upload-frontend.yml

```diff
@@ -0,0 +1,38 @@
+name: Upload Frontend to secure pipelines
+
+on:
+  workflow_call:
+    inputs:
+      environment: { required: true, type: string }
+      artifact-name: { required: true, type: string }
+
+concurrency: deploy-product-frontend-secure-pipelines-${{ inputs.environment }}
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+
+    environment:
+      name: ${{ inputs.environment }}
+      url: ${{ steps.deploy.outputs.pipeline-url }}
+
+    steps:
+      - name: Deploy Product
+        id: deploy-product
+        uses: alphagov/di-github-actions/secure-pipelines/deploy-fargate@407381405d22479808a226de506ad19ba22fd3f5
```

Comment:
This should be govuk-one-login not alphagov

Reply:
fixed